.docker/opt/configure-aiida.sh

```diff
@@ -42,7 +44,7 @@ if [[ ${NEED_SETUP_PROFILE} == true ]]; then
         --scheduler core.direct                                    \
         --work-dir /home/aiida/aiida_run/                          \
         --mpirun-command "mpirun -np {tot_num_mpiprocs}"           \
-        --mpiprocs-per-machine 1 &&                                \
+        --mpiprocs-per-machine ${MPI_PROCS_PER_MACHINE:-${cpu_count}} && \
```

Comment:
Maybe rename `MPI_PROCS_PER_MACHINE` to `LOCALHOST_MPI_PROCS_PER_MACHINE` to make it more specific, just in case there ever is another machine to be configured

Reply:
That seems like a premature generalization to me.

Reply:
It is not so much just generalization, but mostly for clarity by being more explicit.

Reply:
DONE.